packages/react-components/react-color-picker-preview/stories/src/ColorPicker/ColorPickerPopup.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { tinycolor } from '@ctrl/tinycolor';
+import { makeStyles, Button, Popover, PopoverSurface, PopoverTrigger } from '@fluentui/react-components';
+import {
+  ColorPicker,
+  ColorSlider,
+  AlphaSlider,
+  ColorPickerProps,
+  ColorArea,
+} from '@fluentui/react-color-picker-preview';
+
+const useStyles = makeStyles({
+  example: {
+    width: '300px',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '10px',
+  },
+  previewColor: {
+    margin: '10px 0',
+    width: '50px',
+    height: '50px',
+    borderRadius: '4px',
```

Comment:
won't be better to have this value in token? (borderRadiusMedium). I feel it might be useful, if the users will check the source code of docs and copy/paste something, then the correct token is already present 🙂

Reply:
yeah, thanks for catching this